app/components/RouteTabs.tsx

```diff
@@ -39,20 +39,30 @@ export interface RouteTabsProps {
   children: ReactNode
   fullWidth?: boolean
   sideTabs?: boolean
+  tabListClassName?: string
 }
 /** Tabbed views, controlling both the layout and functioning of tabs and the panel contents.
  *  sideTabs: Whether the tabs are displayed on the side of the panel. Default is false.
  */
-export function RouteTabs({ children, fullWidth, sideTabs = false }: RouteTabsProps) {
+export function RouteTabs({
+  children,
+  fullWidth,
+  sideTabs = false,
+  tabListClassName,
+}: RouteTabsProps) {
   const wrapperClasses = sideTabs
     ? 'ox-side-tabs flex'
     : cn('ox-tabs', { 'full-width': fullWidth })
   const tabListClasses = sideTabs ? 'ox-side-tabs-list' : 'ox-tabs-list'
-  const panelClasses = cn('ox-tabs-panel', { 'flex-grow': sideTabs })
+  const panelClasses = cn('ox-tabs-panel @container', { 'ml-5 flex-grow': sideTabs })
```

Comment:
Container query lets us move the datepicker onto the next line when we start running out of space. Flex wrap might work here, but eventually I think I'd like to do something cleverer with the layout when it reflows.